From 7c685d14199e51667996625323add06bc97cf08f Mon Sep 17 00:00:00 2001
From: Antoine Belvire <antoine.belvire@opensuse.org>
Date: Wed, 16 May 2018 08:01:39 +0200
Subject: [PATCH] Fix mime-info format

update-mime-info may fail to parse the file if attributes are not
between quotes.

This commit put attributes between quotes, fix the indentation and
remove a typo (unneeded quote at the end of a line).
---
 resources/powdertoy-save.xml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/resources/powdertoy-save.xml b/resources/powdertoy-save.xml
index bfaeea6b..6e54fd75 100644
--- a/resources/powdertoy-save.xml
+++ b/resources/powdertoy-save.xml
@@ -1,8 +1,8 @@
 <?xml version="1.0"?>
-	<mime-info xmlns='http://www.freedesktop.org/standards/shared-mime-info'>
+<mime-info xmlns='http://www.freedesktop.org/standards/shared-mime-info'>
 	<mime-type type="application/vnd.powdertoy.save">
 		<comment>Powder Toy save</comment>
-		<glob pattern=*.cps/>
-		<glob pattern=*.stm/>
-	</mime-type>"
+		<glob pattern="*.cps"/>
+		<glob pattern="*.stm"/>
+	</mime-type>
 </mime-info>
-- 
2.16.3

